Text/item_text.txt

```diff
@@ -24,10 +24,10 @@ A mysterious gem giving its[NL]
 wielder special ability. +10 crit.[X]
 
 ## nameDragonshard
-Dragonshard[X]
+Wyvernstone[X]
```

Comment:
Why not just Dragonshard? Wyvernstone has a bit of an implication that they're transforming into wyverns. Which is a bit awkward since those are common mounts; kinda incomparable to dragons.

Reply:
because the icon alusq made all pretty-like was a full stone and "dragonstone" and "dragonstone" is obviously wrong to do to a poor player